csharp/src/Microsoft.ML.OnnxRuntime/OrtEnv.shared.cs

```diff
@@ -7,6 +7,14 @@
 
 namespace Microsoft.ML.OnnxRuntime
 {
+    public enum OrtCompiledModelCompatibility
```

Comment:
Wasn't too sure about placement of this one. Advice welcome.

Reply:
I could find a better place, should be fine.